mobile/lib/modules/home/views/home_page.dart

```diff
@@ -121,8 +114,10 @@ class HomePage extends HookConsumerWidget {
                 ),
               ),
             ),
-            _buildSelectedItemCountIndicator(),
-            _buildBottomAppBar(),
+            if (isMultiSelectEnable) ...[
```

Comment:
what is the spread operator at the beginning of the list? I am familiar with the syntax `[...items, newItem]` but `...[]` is new to me

Reply:
[] is list, and items too.
So [...items, newItem] is same [...[], newItem].